x/auth/ante/fee.go

```diff
@@ -25,7 +25,7 @@ type DeductFeeDecorator struct {
 
 func NewDeductFeeDecorator(ak AccountKeeper, bk types.BankKeeper, fk FeegrantKeeper, tfc TxFeeChecker) DeductFeeDecorator {
 	if tfc == nil {
-		tfc = checkTxFeeWithValidatorMinGasPrices
+		tfc = CheckTxFeeWithValidatorMinGasPrices
```

Comment:
I think overall adding a new argument to `NewDeductFeeDecorator` is a better API

Reply:
Do you mean just make a custom implementation of `TxFeeChecker` right? Do you thin the default one is good enough to act as a default, or it shouldn't be used in production at all?

Reply:
Yes, exactly. It's not even writing your own decorator, but your own `TxFeeChecker`, which is like 30LOC. IMO the SDK default one is being used in production already, so is good enough.

Reply:
ok, I'll close it.